src/z3c/checkversions/base.py

```diff
@@ -105,10 +101,8 @@ def check(self, level=0):
                     continue
                 # trunk the version tuple to the first `level` elements
                 # (and remove *final and pad both to the level length)
-                trunked_current = [x for x in parsed_version[:level]
-                                   if not x.startswith('*')]
-                trunked_candidate = [x for x in dist.parsed_version[:level]
-                                     if not x.startswith('*')]
+                trunked_current = parsed_version.base_version.split('.')[:level]
+                trunked_candidate = dist.parsed_version.base_version.split('.')[:level]
```

Comment:
The comment needs some updating -- the "remove *final" bit is no longer true.

Reply:
Good catch!